mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -629,7 +629,7 @@ public void searchForMessageShouldReturnMessagesFromMyDelegatedMailboxes() throw
         mailboxManager.setRights(delegatedMailboxPath,
             MailboxACL.EMPTY.apply(MailboxACL.command()
                 .forUser(USER_1)
-                .rights(MailboxACL.Right.Read)
+                .rights(MailboxACL.Right.Read, MailboxACL.Right.Lookup)
```

Comment:
We miss unit test for this change

Reply:
I reverted it after the last code changes I propose.